lib/entities/integrations/models.ts

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+import {IntegrationType} from "./enums";
+
+export type IIntegrationType = {
+    type: IntegrationType
+    value: string;
+};
+
+export interface IIntegrationData { }
+
+export interface IWebhooksIntegrationData extends IIntegrationData {
```

Comment:
Because this isn't in use, I was thinking we add this as an or to the `IIntegration.data` and see how that works. When we have more integrations, we just concat it on with the or symbol

Reply:
Looks like there was some confusion with the types here. Have discussed this now. We can get rid of the unused models and simplify the approach as so - 

```
export interface IWebhooksIntegrationData {
  url: string;
}

export type IIntegration = {
  identifier: string;
  name: string;
  type: IntegrationType;
  data: IWebhooksIntegrationData;
};

```